optuna/study.py

```diff
@@ -149,6 +149,15 @@ def trials(self) -> List[FrozenTrial]:
 
         return self.get_trials(deepcopy=True, states=None)
 
+    def _is_multi_objective(self) -> bool:
+        """Return True if the study has multi objectives.
```

Comment:
```suggestion
        """Return True if the study has multiple objectives.
```

I think it would be better to avoid an abbreviation.